introspection-engine/connectors/introspection-connector/src/lib.rs

```diff
@@ -98,16 +99,7 @@ impl IntrospectionContext {
 
         let previous_schema_config_only = psl::parse_schema(config_blocks).unwrap();
 
-        Self::new_naive(previous_schema_config_only, composite_type_depth)
-    }
-
-    fn new_naive(previous_schema: psl::ValidatedSchema, composite_type_depth: CompositeTypeDepth) -> Self {
-        IntrospectionContext {
-            previous_data_model: psl::dml::Datamodel::new(),
-            previous_schema,
-            composite_type_depth,
-            render_config: true,
-        }
+        Self::new(previous_schema_config_only, composite_type_depth)
     }
```

Comment:
Simplifications that became possible from not having re_introspection.rs anymore.